CHANGES.md

```diff
@@ -216,6 +218,7 @@ Released on August 26, 2022.
 [#2219]: https://github.com/planetarium/libplanet/pull/2219
 [#2227]: https://github.com/planetarium/libplanet/pull/2227
 [#2228]: https://github.com/planetarium/libplanet/pull/2228
+[#2229]: https://github.com/planetarium/libplanet/pull/2229
```

Comment:
Please move your changelog to To be released section.

Reply:
Thanks for letting me know. i move v0.41.0 to v0.42.0.